app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
```

Comment:
if you are planing to use this variable ```_``` it is required good naming (mostly ```_``` use when variable is not planing to be used)

also you can iterate directly by argument
```
for arg in argv:
```